BUILD

```diff
@@ -292,3 +315,19 @@ upgrading GCC. If you have libstdc++ corresponding to GCC 5.x or 6.x, you *must*
 old ABI, but Dinit will be broken if you upgrade to GCC 7. If you have libstdc++ from GCC 7, you
 *must* build with the new ABI. If the wrong ABI is used, Dinit may still run successfully but any
 attempt to load a non-existing service, for example, will cause Dinit to crash.
+
+
+Special note for the Clang compiler and the Libcxxrt C++ runtime
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
+
+Clang with the Libcxxrt C++ runtime library on certain platforms has an issue that prevents
```

Comment:
Clang _when used as a compiler together_ with the Libcxxrt C++ runtime library ...